includes/admin/class-ajax.php

```diff
@@ -851,22 +846,22 @@ public function erp_reset_data() {
 
         $reset_text = sanitize_text_field( wp_unslash( $_POST['erp_reset_confirmation'] ) );
 
-        if ( ! isset( $_POST['erp_reset_confirmation'] ) || 'Reset' !== sanitize_text_field( wp_unslash( $_POST['erp_reset_confirmation'] ) ) ) {
-            $this->send_error( esc_html__( 'Invalid confirmation text. Please give valid confirmation text.', 'erp' ) );
-        }
+        if ( $reset_text === 'Reset' ) {
```

Comment:
Why did you have to refactor this part? Please don't do any unnecessary refactoring. Restore the previous implementation here and just remove the line#847 (`$reset_text = sanitize_text_field( wp_unslash( $_POST['erp_reset_confirmation'] ) );`)